packages/components/src/styles/chip/chip.stories.ts

```diff
@@ -37,16 +37,16 @@ export const Default = {
   }
 };
 /**
- * Use `sd-chip` modifiers for alternative appearances:
- * - `--primary-200` (default)
- * - `--primary-300`
- * - `--primary-500`
- * - `--white`
+ * Use the `sd-chip` classes for alternative appearances:
+ * - primary-200 is the default background color
```

Comment:
```suggestion
 * - `primary-200` is the default background color
```

Reply:
In this case, me and @MartaPintoTeixeira believe it shouldn't be highlighted because it is not a class or a value. Therefore, it serves as a way to differentiate. We think the highlight may mislead the user into believing it is also a class.